src/models/drag-event-counter.spec.ts

```diff
@@ -66,6 +66,7 @@ describe('Model: DragEventCounter', () => {
     dragEventCounter.inc('a1');
     dragEventCounter.inc('a1');
     dragEventCounter.inc('a2');
+    // tslint:disable-next-line:no-magic-numbers
     expect(dragEventCounter.dragEventCountMap['a1']).toEqual(2);
```

Comment:
One idea was to spy on dragEventCount.inc and use the call count as the toEqual() value, but I did not manage to make it work...